auditbeat/cmd/show_linux.go

```diff
@@ -63,7 +63,7 @@ func init() {
 		},
 	}
 	showStatus.Flags().BoolVarP(&singleLineStatus, "single-line", "s", false, "Output status as a single line")
-	cmd.ShowCmd.AddCommand(&showRules, &showStatus)
```

Comment:
Why did show_linux.go need to be changed? Asking b/c I probably don't understand something. It was only adding itself to the `github.com/elastic/beats/v7/auditbeat/cmd.ShowCmd` which is isolated to the Auditbeat RootCmd. I don't see where this would cause a conflict under agentbeats combined command.

Reply:
Because it causes a circular import of cmd now, because the include had to be moved from main.go into root.go.

Without this change it will not compile. Even auditbeat will not compile without this change.

Reply:
Oh, yes I see it now. That's very subtle.